packages/mds-agency/utils.ts

```diff
@@ -127,7 +127,7 @@ export async function getVehicles(
   logger.info(`read ${total} deviceIds in /vehicles`)
 
   const events = rows.length > 0 ? await cache.readEvents(rows.map(record => record.device_id)) : []
-  const eventMap: { [s: string]: VehicleEvent } = {}
+  const eventMap: { [s: string]: VehicleEvent | undefined } = {}
```

Comment:
It can be undefined but it can't be null?

Reply:
Yeah, this is to handle cache misses!